RRemoteConfig/APIClient.swift

```diff
@@ -17,28 +17,28 @@ internal class APIClient {
         self.session = session
     }
 
-    func send<T>(request: URLRequest, decodeAs: T.Type, completionHandler: @escaping (Result<Any, Error>) -> Void) where T: Decodable {
+    func send<T>(request: URLRequest, decodeAs: T.Type, completionHandler: @escaping (Result<Any, Error>, _ httpResponse: HTTPURLResponse?) -> Void) where T: Decodable {
```

Comment:
I don't like that I had to expose the response in the completion handler here but as this is a generic function it seemed like the most generic way to do it.